pom.xml

```diff
@@ -37,7 +37,7 @@
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
         <!--The common Hadoop version for Spark 2.4.7 and 3.0.1, which lakeFS supports-->
-        <hadoop.version>2.7.7</hadoop.version>
+        <hadoop.version>2.9.2</hadoop.version>
```

Comment:
Comment is no longer quite as correct, so consider getting rid of it.

Also please make sure to document _why_ we need Hadoop >2.7 here (**not** in this PR, but maybe just open an issue to do it).